src/PortingAssistantVSExtensionClient/Commands/ProjectPortingCommand.cs

```diff
@@ -104,6 +104,11 @@ private async void Execute(object sender, EventArgs e)
                 if (!CommandsCommon.SetupPage()) return;
                 string SelectedProjectPath = SolutionUtils.GetSelectedProjectPath();
                 selectedProjectName = Path.GetFileName(SelectedProjectPath);
+                if (!UserSettings.Instance.SolutionAssessed)
+                {
+                    NotificationUtils.ShowInfoMessageBox(this.package, "please run a full assessment before porting", "");
```

Comment:
Message should be: "Please run a full assessment before porting" (capitalize first letter)